backend/de.metas.ui.web.base/src/main/sql/postgresql/ddl/views/QtyDemand_QtySupply_V.sql

```diff
@@ -3,49 +3,116 @@ DROP VIEW IF EXISTS QtyDemand_QtySupply_V
-      GROUP BY m_shipmentschedule.ad_client_id, m_shipmentschedule.ad_org_id, m_shipmentschedule.m_warehouse_id,
-               m_shipmentschedule.m_product_id, attributesKey, product.c_uom_id) AS sched
-         FULL OUTER JOIN (SELECT rs.ad_client_id,
-                                 rs.ad_org_id,
-                                 rs.m_warehouse_id,
-
-                                 -- make sure that m_receiptschedule.qtyToMove is also in the product's UOM
-                                 SUM(uomconvert(rs.m_product_id, rs.c_uom_id, product.c_uom_id, rs.qtyToMove)) AS qtyToMove,
-
-                                 rs.m_product_id,
-                                 generateasistorageattributeskey(rs.m_attributesetinstance_id)                    attributesKey,
-                                 product.c_uom_id
-                          FROM m_receiptschedule rs
-                                   INNER JOIN m_product product ON rs.m_product_id = product.m_product_id
-                          GROUP BY rs.ad_client_id, rs.ad_org_id,
-                                   rs.m_product_id, rs.m_warehouse_id,
-                                   attributesKey,
-                                   product.c_uom_id) AS rs
-                         ON rs.m_product_id = sched.m_product_id
-                             AND rs.ad_client_id = sched.ad_client_id
-                             AND rs.ad_org_id = sched.ad_org_id
-                             AND rs.m_warehouse_id = sched.m_warehouse_id
-                             AND rs.attributesKey::text = sched.attributesKey::text
-                             AND rs.c_uom_id = sched.c_uom_id
-;
\ No newline at end of file
+SELECT t.ad_client_id,
+       t.ad_org_id,
+       p.name                                                                                                                  AS ProductName,
+       p.value                                                                                                                 AS ProductValue,
+       p.m_product_id,
+       p.c_uom_id,
+       t.attributesKey,
+       t.m_warehouse_id,
+       SUM(qtyReserved)                                                                                                        AS qtyReserved,
+       SUM(qtyToMove)                                                                                                          AS qtyToMove,
+       SUM(qtyToProduce)                                                                                                       AS qtyToProduce,
+       SUM(qtyForecasted)                                                                                                      AS qtyForecasted,
+       SUM(qtyStock)                                                                                                           AS qtyStock,
+       ROW_NUMBER() OVER (ORDER BY t.ad_client_id, t.ad_org_id, p.m_product_id, p.c_uom_id, t.attributesKey, t.m_warehouse_id) AS QtyDemand_QtySupply_V_ID
```

Comment:
pls no row number as ID

Reply:
I have no idea what to use as ID, and is mandatory if we want processes on the page. Any suggestion?